protocol/contracts/templegold/DaiGoldAuction.sol

```diff
@@ -0,0 +1,262 @@
+    }
+
+    /**
+     * @notice Checkpoint total Temple Gold distributed for next epoch.
+     * Can only be called by Temple Gold contract
+     * @param amount Amount of Temple Gold to checkpoint
+     */
+    function notifyDistribution(uint256 amount) external override {
+        if (msg.sender != address(templeGold)) { revert CommonEventsAndErrors.InvalidAccess(); }
+        /// @notice Temple Gold contract mints TGLD amount to contract before calling `notifyDistribution`
+        nextAuctionGoldAmount += amount;
+        lastRewardNotificationTimestamp = uint96(block.timestamp);
+        emit GoldDistributionNotified(amount, block.timestamp);
+    }
+    
+    /**
+     * @notice Get Temple Gold supply for epoch
+     * @param epochId Epoch Id
+     * @return Temple Gold supply
+     */
+    function epochGoldSupply(uint256 epochId) external view override returns (uint256) {
+        return epochs[epochId].totalAuctionTokenAmount;
+    }
+
+    /**
+     * @notice Get current epoch
+     * @return Epoch Id
+     */
+    function currentEpoch() external view override returns (uint256) {
+        return _currentEpochId;
+    }
+
+    /**
+     * @notice Get next epoch
+     * @return Next epoch Id
+     */
+    function nextEpoch() public view override returns (uint256) {
+        return _currentEpochId + 1;
+    }
+
+    /**
+     * @notice Check if current epoch ended. That is, if current auction ended.
+     * @return Bool if epoch ended
+     */
+    function isCurrentEpochEnded() external view override returns (bool) {
+        return _isCurrentEpochEnded();
+    }
+
+    /**
+     * @notice Check if current auction epoch is allowing bid deposits
+     * @return Bool if allowing deposit
+     */
+    function canDeposit() external view override returns (bool) {
+        return _canDeposit();
+    }
+
+    /**
+     * @notice Get auction configuration
+     * @return Auction configuration
+     */
+    function getAuctionConfig() external override view returns (AuctionConfig memory) {
+        return auctionConfig;
+    }
+
+    /**
+     * @notice Get claimable amount for an epoch
+     * @dev For current epoch, function will return claimable at current time. This can change with more user deposits
+     * @param depositor Address to check amount for
+     * @return Claimable amount
+     */
+    function getClaimbaleAtCurrentTimestamp(address depositor) external override view returns (uint256) {
+        return getClaimableAtCurrentTimestamp(depositor, _currentEpochId);
+    }
+
+    /**
+     * @notice Get claimable amount for an epoch
+     * @dev For current epoch, function will return claimable at current time. This can change with more user deposits
+     * @param depositor Address to check amount for
+     * @param epochId Epoch id
+     * @return Claimable amount
+     */
+    function getClaimableAtCurrentTimestamp(address depositor, uint256 epochId) public override view returns (uint256) {
```

Comment:
Should this be called `getClaimableAtEpoch()` given you're passing the epoch id in?

And the one above `getClaimableAtCurrentEpoch()`